src/Altinn.Notifications.Email.Core/Status/EmailSendResultMapper.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Altinn.Notifications.Email.Core.Status;
+
+/// <summary>
+/// Mapper handling parsing to EmailSendResult
+/// </summary>
+public static class EmailSendResultMapper
+{
+    /// <summary>
+    /// Parse AcsEmailDeliveryReportStatus to EmailSendResult
+    /// </summary>
+    /// <param name="deliveryStatus">Delivery status from Azure Communication Service</param>
+    /// <returns></returns>
+    /// <exception cref="ArgumentException">Throws exception if unknown delivery status</exception>
+    public static EmailSendResult ParseDeliveryStatus(string? deliveryStatus)
+    {
+        switch (deliveryStatus)
+        {
```

Comment:
Wondering how we persist the information about an unknown status being pushed to us. Should we log the operationId and status so we can follow up and potentially manually update the database? Currently I guess only the status is logged, but we won't know for which sms

Reply:
Added logging in the controller